salt/orchestrate/aws/mitxpro_autoscaling.sls

```diff
@@ -1,6 +1,10 @@
 {% for app_name in ['edxapp', 'edx-worker'] %}
 {% set env_settings = salt.cp.get_url("https://raw.githubusercontent.com/mitodl/salt-ops/main/salt/environment_settings.yml", dest=None)|load_yaml %}
 {% set ENVIRONMENT = salt.environ.get('ENVIRONMENT', 'mitxpro-production') %}
+
+# Using QA AMI build to deploy production instances 
+{% set AMI_ENV = salt.environ.get('AMI_ENV', 'mitxpro-qa') %}
```

Comment:
Don't forget that we need to manually set the AMI ID in the SDB in production.

Reply:
Why does that matter since we're passing the AMI to the launch configuration

Reply:
Because it's trying to pull the ID from SDB, and the SDB systems in production and QA can't talk to each other, so we need to update the value in the production SDB for it to get fetched properly.

Reply:
The only place I see it using the sdb is for the `release_number` and that appears useless as it's not used anywhere in that file. As far as the AMI, I only see it using the AMI name and uses that to create the launch config. What am I missing?

Reply:
Nevermind, I got it

Reply:
Nevermind, I was thinking about how it's used in the residential orchestrate, didn't realize it was doing a lookup of the actual AMI.